### ISSUE TITLE
sentence navigation command Alt + down arrow doesn't work in Microsoft Word

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5177]
**Reported by bhavyashah on 2015-06-22 13:52**
I am using NVDA version 2015.2 stable, Microsoft Office Word 2007 on a Windows 7 Ultimate system. Reproduction rate is 100% consistency till now.
This issue is extremely strange, more so, because one in the pair works functionally while the second in the pair doesn't and has some other peculiar consequences as well. More specifically, this is an issue with the sentence navigation command Alt + down arrow to read the next sentence.
When I press Alt + up arrow, NVDA reads the previous sentence as expected, without any hitch or problem.
When I press Alt + down arrow, NVDA doesn't read the next sentence as expected. Instead, NVDA freezes for several seconds, and, automatically comes back to life when I issue further speech invoking commands.
A strange discovery I've made right now while trying to reproduce the issue again to describe it here appropriately...
I opened a new blank Word document, and typed three sentences, where both sentence navigation commands functioned perfectly as expected. I was startled, and, I opened one of my textbooks Word document, where I've reproduced this issue with complete consistency, and, I did it once again as well. My guess is that, Alt + down arrow results in a temporary freeze only in long Word documents and not short ones. Regardless, my assumption is likely to be incorrect, and, hence, I shall attach the document with which I have reproduced this issue many times for your review and thought.

**Comment by leonardder** [comment]
cc @bhavyashah

**Comment by bhavyashah** [comment]
This is extremely weird. Using NVDA 2017.3 RC 1 and Word 2010 on a Windows 8.1 system, below recorded are my results:
* Short document: Both Alt+up and down arrow work as expected.
* Long document: Alt+up and down arrow does just nothing. In this document containing over 600,000 characters (with space), pressing Alt+up and down arrow, both in focus and browse mode, has no effect. NVDA neither freezes nor navigate to the previous or next sentence. Set at Debug, the logg contains no useful information either.
Can others please test and report whether this oddity was reproducible at their end?

**Comment by ruifontes** [comment]
Tested with 435000 characters and Word 2013, and it performs as expected...

Rui


-----Mensagem Original----- 
De: bhavyashah
Data: 18 de agosto de 2017 08:30
Para: nvaccess/nvda
Cc: Subscribed
Assunto: Re: [nvaccess/nvda] sentence navigation command Alt + down arrow 
doesn't work in Microsoft Word (#5177)



This is extremely weird. Using NVDA 2017.3 RC 1 and Word 2010 on a Windows 
8.1 system, below recorded are my results:

Short document: Both Alt+up and down arrow work as expected.
Long document: Alt+up and down arrow does just nothing. In this document 
containing over 600,000 characters (with space), pressing Alt+up and down 
arrow, both in focus and browse mode, has no effect. NVDA neither freezes 
nor navigate to the previous or next sentence. Set at Debug, the logg 
contains no useful information either.
Can others please test and report whether this oddity was reproducible at 
their end?

**Comment by bhavyashah** [comment]
Update: Even with Word 2016, NVDA 2018.3 Beta and Windows 8.1, I can reproduce this issue with sentence navigation in a large document. However, not akin to the previously described behaviour, pressing Alt+up/down arrow did seem to change the focus which I ascertained by reading the current line after each Alt+up/down arrow press, but NVDA still does not verbalize it then and there. Now, we do have a separate Sentence Nav NVDA add-on also, but which I have not personally tried and therefore cannot comment on.

**Comment by netblue44** [comment]
I can also conform me

**Comment by Adriani90** [comment]
I cannot reproduce this in a document with 241531 characters in NVDA alpha-20097,6a573cd9. Tested in MS Word 2010 and MS Word 365. Closing as works for me, but please feel free to comment in case you are still having this issue and we can reopen the discussion.
